src/components/ec-full-screen-overlay/ec-full-screen-overlay.vue

```diff
@@ -0,0 +1,105 @@
+<template>
```

Comment:
We need a placeholder for the content we want to put into the component and provide a `<slot>` that we will populate from the EBO.